src/components/article-card.jsx

```diff
@@ -0,0 +1,18 @@
+ const ArticleCard = ({articles}) => {
```

Comment:
I would have this components folder nested within the src folder.   (Ignore the placing of this comment)